packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -243,7 +243,7 @@ function _DevtoolsView(props: _DevtoolsViewProps): React.ReactElement {
 	}, [messageRelay, setContainers]);
 
 	return (
-		<Stack enableScopedSelectors horizontal styles={stackStyles}>
+		<Stack enableScopedSelectors horizontal styles={stackStyles} id="fluid-devtools-view">
```

Comment:
Where is this used?

Reply:
It isn't currently. This could be removed.

Reply:
```suggestion
		<Stack enableScopedSelectors horizontal styles={stackStyles}>
```